crates/runner-host/crates/web-client/src/main.rs

```diff
@@ -18,7 +18,27 @@ mod board_components;
 mod console;
 mod hooks;
 
+use cfg_if::cfg_if;
+
 fn main() {
-    wasm_logger::init(wasm_logger::Config::default());
+    wasm_logger::init(wasm_logger::Config::new(get_log_level()));
```

Comment:
This could simply be something like this:
```suggestion
    #[cfg(feature = "log_trace")]
    wasm_logger::init(wasm_logger::Config::new(log::Level::Trace));
    #[cfg(feature = "log_debug")]
    wasm_logger::init(wasm_logger::Config::new(log::Level::Debug));
    #[cfg(feature = "log_info")]
    wasm_logger::init(wasm_logger::Config::new(log::Level::Info));
    #[cfg(feature = "log_warn")]
    wasm_logger::init(wasm_logger::Config::new(log::Level::Warn));
    #[cfg(feature = "log_error")]
    wasm_logger::init(wasm_logger::Config::new(log::Level::Error));
```

Then we don't need the `cfg_if` dependency and when no features are selected, logging is disabled.